app/Repositories/H5pLibrary/H5pLibraryRepositoryInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Repositories\H5pLibrary;
+
+use Djoudi\LaravelH5p\Eloquents\H5pLibrary;
+use App\Repositories\EloquentRepositoryInterface;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+interface H5pLibraryRepositoryInterface extends EloquentRepositoryInterface
```

Comment:
Any public functions defined in H5pLibraryRepository, should be placed in this interface.

Reply:
Placed public functions defined in H5pLibraryRepository and ActivityRepository in their respective interfaces.